src/cmap/stream_description.ts

```diff
@@ -73,4 +80,12 @@ export class StreamDescription {
       this.compressor = this.compressors.filter(c => response.compression?.includes(c))[0];
     }
   }
+
+  /* @internal */
+  parseServerConnectionID(serverConnectionId: number | Double | bigint | Long): bigint {
+    return Long.isLong(serverConnectionId)
+      ? serverConnectionId.toBigInt()
+      : // @ts-expect-error: Doubles are coercible to number
```

Comment:
Can we also add a comment (and maybe link to the server code that supports this) that connection ids are always integral, and so coercing a double to an integer type is safe?

Reply:
I added the comment but did not link to server code as it's location can change over time.